content/knowledge-base/firebase-crashlytics-dsym-uploading.md

```diff
@@ -41,6 +41,22 @@ In order to upload the dSYM files to Firebase Crashlytics, add the following scr
     $CM_BUILD_DIR/ios/Pods/FirebaseCrashlytics/upload-symbols -gsp ios/Runner/GoogleService-Info.plist -p ios $dsymPath
   fi
   ```
+ 
+The above-mentioned **dsymPath** is Flutter specific and it could change depending on what platform the app is built. 
+
+With React Native applications:
+
+```
+dsymPath=$(find $CM_BUILD_DIR/build/ios/xcarchive/*.xcarchive -name "*.dSYM" | head -1)
+```
+
+With Native iOS applications:
+
+```
+dsymPath=$(find $CM_BUILD_DIR/build/ios/xcarchive/*.xcarchive -name "*.dSYM" | head -1)
+```
```

Comment:
Why repeat the same line of script twice?

Put something like "With React Native or Native iOS applications:" then put the script.

Reply:
True that, changed!